packages/design-system/src/components/Spinner/Spinner.tsx

```diff
@@ -47,14 +47,15 @@ export const Spinner: React.FunctionComponent<SpinnerProps> = (props: SpinnerPro
 
   return (
     <span className={className} role={props.role}>
-      <span className="ds-u-visibility--screen-reader">{props['aria-valuetext']}</span>
+      <span className="ds-u-visibility--screen-reader">
+        {props['aria-valuetext'] ?? t('spinner.ariaText')}
```

Comment:
reverting the spinner update - dynamically updating prop isn't supported when defined as defaultProp